src/components/consul/detail/ConsulTagListItem.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { RegularText } from '../../Text';
+import { colors } from '../../../config';
+
+export const ConsulTagListItem = ({ item }) => {
+  const { name } = item.item;
```

Comment:
`item.item` is a bit weird here. could this nesting be optimized or maybe named differently?

Reply:
The code has been updated. Item.item in all list items has been removed. New names were given.